Dockerfile

```diff
@@ -0,0 +1,11 @@
+
+FROM ubuntu:16.04
+
+
+COPY . /sortmerna
```

Comment:
This will compile whatever version you just have checked out. Note that the COPY statement copies the content of the current directory into the container. Assuming you just do a "git clone", it will copy the latest version of the master branch into the container and create an image from that. But you can also use "git checkout ..." to create an Docker image for other versions of sortmerna if you want.